developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,403 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FAs) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible units, which can represent different assets, such as currency, shares, material in games, or many other types of assets. Furthermore, fungible assets can be used to build decentralized applications that require a token economy, such as decentralized exchanges or gaming platforms.
+
+The [fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move) provides a standard, type-safe framework for defining fungible assets within the Aptos Move ecosystem.
+
+In this standard, fungible assets are stored in `Object<FungibleStore>` that has a specific amount of units. Fungible assets are units that are interchangeable with others of the same metadata. The standard is built upon object model so all the resources defined here are included in object resource group and stored inside objects. The fungible assets within an object can be divided into smaller units creating new stores without creating new units. Similarly they can be combined to aggregate units into fewer objects. The standard also supports minting new units and burning existing units with appropriate controls.
```

Comment:
"Fungible assets are units that are interchangeable with others of the same metadata. The standard is built upon object model so all the resources defined here are included in object resource group and stored inside objects"

This sentence can also be rewritten to be more understandable: "Fungible assets have a metadata object define that include information such as name, symbol and decimals. Fungible assets of a specific metadata type can be stored in objects that contain a FungibleStore resource" (this flows into the sentence from the earlier comment)